### PR TITLE
LPS-126345

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by LocalizedNestedFieldsWebContent#DefaultValuesDisplayedForNonTranslatedWebContent
See https://github.com/brianchandotcom/liferay-portal/pull/98417

**Comment by emanuelfisilva** [comment]
Thanks for your review @Tim-Cao , but we'll choose to keep these validations separated.

File: portal-web/test/functional/com/liferay/portalweb/macros/CKEditor.macro [hunk 1]
```diff
@@ -117,4 +117,15 @@ definition {
 		SelectFrame(value1 = "relative=top");
 	}
 
+	macro assertContentIframe{
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/CKEditor.macro]: Use existing macro

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3769
View total diff https://github.com/liferay-echo/liferay-portal/pull/3769/commits/6898d2bd161db34faca6940a74b2d602a9626459

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**